### ISSUE TITLE
Issue with Math operations

### ISSUE THREAD

**Issue by paslavsky** [anboralabs/intellij-firebase-highlighter#56]
The rule file:
```
rules_version = '2';
service cloud.firestore {
    match /databases/{database}/documents {
        match /{document=**} {
            allow read, write: if false;
        }

        function testMath() {
            let a = 1;
            let b = 1;
            let c = a + b;
            return 1 + b == c;
        }
    }
}
```
Errors: 
* line 11: `'(', '.', ';', <boolean operator>, LET_KEYWORD, RETURN_KEYWORD or '[' expected, got '+'`
* line 12: `number expected, got 'b'`

The plugin shows the error when storing the math operation results into the variable using already existing variables.

The following examples works very well:
* `return 1 + 2 == c;`
* `return 1 + 1 == 2;`

**Comment by Artelis** [comment]
Came here to report the same thing!

```
function sessionHasExpired() {
    return (request.auth.token.auth_time + (60 * 60 * 24 * 30)) * 1000 > request.time.toMillis();
}
```

The plugin adds an error underline at the space between "auth_time" and "+" with the same `'(', '.', ';', <boolean operator>, LET_KEYWORD, RETURN_KEYWORD or '[' expected, got '+'` as above.

**Comment by dalgarins** [comment]
<img width="584" alt="Captura de Pantalla 2021-04-12 a la(s) 11 15 07 p  m" src="https://user-images.githubusercontent.com/8527174/114496175-ffb78c80-9be4-11eb-842e-1fe80ac54f9b.png">

<img width="706" alt="Captura de Pantalla 2021-04-12 a la(s) 11 15 28 p  m" src="https://user-images.githubusercontent.com/8527174/114496194-05ad6d80-9be5-11eb-9dda-0e6affaadd94.png">

@Artelis and @paslavsky 

Thanks for testing, issue will be fixed on version: 2.5.3
